tests/cases/standalone/common/promql/scalar.sql

```diff
@@ -101,4 +101,46 @@ TQL EVAL (0, 15, '5s') scalar(scalar(host{host="host2"}) + 1);
 
 TQL EVAL (0, 15, '5s') scalar(scalar(host{host="host1"}) + scalar(host{host="host2"}));
 
+-- Test clamp functions with vector input and scalar bounds
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') clamp(host, 0, 12);
```

Comment:
add one for `TQL EVAL (0, 15, '5s') clamp(host, 6-6, 6+6);` and one for `TQL EVAL (0, 15, '5s') clamp(host, 12, 0);`